specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/firewallPolicy.json

```diff
@@ -0,0 +1,952 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2019-06-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [{
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/firewallPolicies/{firewallPolicyName}": {
+      "delete": {
+        "tags": [
+          "firewallPolicies"
+        ],
+        "operationId": "firewallPolicies_Delete",
```

Comment:
Please start operationId with capital letter for the sake of consistency (for this and the following operations)

Reply:
Sematic validation tests fail with this change. Reverting to lower case

Reply:
I meant that `firewallPolicies_Delete` should start with uppercase letter

Reply:
got it. Thanks for clarifying. Done.